app/istqb-ctel-itp-itpi-complete-guide/istqb-ctel-itp-itpi-complete-guide.css

```
@@ -0,0 +1,961 @@
+/* ─── CSS CUSTOM PROPERTIES ─── */
+            
+
+            /* Scanline overlay */
+            &::before {
```

Stylelint (17.12.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)